src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -249,6 +250,14 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
   const existingPostRun = moduleConfig.postRun || [];
   (moduleConfig as any).preloadPlugins = [];
 
+  var totalResources = Object.keys(resources.assembly).length + Object.keys(resources.pdb || {}).length + /* dotnet.wasm */ 1;
+  if (resourceLoader.bootConfig.resources.runtime.hasOwnProperty('dotnet.timezones.blat'))
+    totalResources++;
+  if (resourceLoader.bootConfig.icuDataMode !== ICUDataMode.Invariant)
+    totalResources++;
```

Comment:
```suggestion
  var totalResources = 
    Object.keys(resources.assembly).length + 
    Object.keys(resources.pdb || {}).length + 
    /* dotnet.wasm */ 1 + 
    resourceLoader.bootConfig.resources.runtime.hasOwnProperty('dotnet.timezones.blat') + 
    (resourceLoader.bootConfig.icuDataMode !== ICUDataMode.Invariant);
```

Not sure how others feel about this approach...